src/hosts/host.entity.mjs

```diff
@@ -46,8 +50,7 @@ export class HostEntity {
   */
   constructor (options) {
     options && Object.assign(this, options)
-
-    this.oid ??= nanoid()
-    this.pid ??= nanoid(128)
+    this.oid ??= generateOID(+config.oid.length)
```

Comment:
The functions returned by `nanoid.customAlphabet` already include the length, so no need to pass them here.

Reply:
If I don't it doesnt work for some reason and hangs.

Reply:
Let me try again

Reply:
Nvm, was bc I wasn't converting env var to int.